### PR TITLE
microcoap_server: add instructions for FF Copper

### PR DESCRIPTION
**Pull request by Lotterleben**
As discussed in https://github.com/RIOT-OS/RIOT/issues/4677: Add instructions on how to make copper work with native. This is my first time using radvd, so I hope this makes sense :D.
Also fix some typos while we're at it.
(I thought it would be nice to add a screenshot of what the firefox window would look like after a successful connection, but I have no idea where to put the image without cluttering the RIOT repo...)

### CODE REVIEW

File: examples/microcoap_server/README.md [hunk 1]
```diff
@@ -58,11 +58,59 @@ Waiting for incoming UDP packet...
...
+=======
+There are multiple external CoAP clients you can use to test the server on native.
+
+libcoap CLI
+-----------
 
 (replace "fe80::f8bf:2bff:fe01:9ea3" with your link-layer address)
 
 ```
 # coap-client coap://[fe80::f8bf:2bff:fe01:9ea3%tap0]/riot/board
```
Comment (cgundogan) [inline examples/microcoap_server/README.md]: Just a hint here: for zsh users it is necessary to put the url in quotes `coap-client "coap://[fe80::f8bf:2bff:fe01:9ea3%tap0]/riot/board"`. If this also works for bashers, then I'd vote to change it in the README.

**Comment by cgundogan** [comment]
The text looks good. ACK from my side (after addressing my comment).
@OlegHahm wasn't it necessary to use a patched version of radvd? If this is still the case, then we should also mention that here.

File: examples/microcoap_server/README.md (hunk 1 above)
Comment (Lotterleben) [inline examples/microcoap_server/README.md]: Yup... Looks good, I'll change that. thanks for the hint :)

**Comment by Lotterleben** [comment]
fwiw, I just used the radvd my package manager provided me with :)

**Comment by cgundogan** [comment]
>fwiw, I just used the radvd my package manager provided me with :)

ah okay .. I think the patch was needed for the 6LoWPAN scenario .. so not on native (:

ACK

**Comment by Lotterleben** [comment]
Ok, all squashed, let's see what Travis says?

**Comment by cgundogan** [comment]
no need to wait for travis here. Changes are only in the README.md => GO

**Comment by Lotterleben** [comment]
Ah I thought there might be some fancy checks I don't know about...

**Comment by OlegHahm** [comment]
> ah okay .. I think the patch was needed for the 6LoWPAN scenario .. so not on native (:

That's correct.

**Pull request merged**